### ISSUE TITLE
when installing java automatically freeze at 100%

### ISSUE THREAD

**Issue by Charlo366** [gorilla-devs/GDLauncher#900]
It first download then extract it but it get stuck on 100% with the green bar and the check icon like it finished but just freeze there.

**Comment by swardhagen** [comment]
Yes, same! If I close the app it looks complete but it isn't and I have to start over again.
